packages/test/local-server-stress-tests/src/ddsModels.ts

```diff
@@ -0,0 +1,203 @@
+			minimizationTransforms,
+		});
+	}
+
+	return modelMap;
+};
+
+export const ddsModelMap = generateSubModelMap(
+	baseMapModel,
+	baseDirModel,
+	baseSharedStringModel,
+	baseIntervalModel,
+);
+
+export interface DDSModelOp {
+	type: "DDSModelOp";
+	channelType: string;
+	channelId: string;
+	op: unknown;
+}
+
+const createDDSClient = (channel: IChannel): DDSClient<IChannelFactory> => {
+	return {
+		channel,
+		containerRuntime: makeUnreachableCodePathProxy("containerRuntime"),
+		dataStoreRuntime: makeUnreachableCodePathProxy("dataStoreRuntime"),
+	};
+};
+
+const covertLocalServerStateToDdsState = (
+	state: LocalServerStressState,
+	channel: IChannel,
+): DDSFuzzTestState<IChannelFactory> => {
+	return {
+		clients: makeUnreachableCodePathProxy("clients"),
+		client: createDDSClient(channel),
+		containerRuntimeFactory: makeUnreachableCodePathProxy("containerRuntimeFactory"),
+		isDetached: state.isDetached,
+		summarizerClient: makeUnreachableCodePathProxy("containerRuntimeFactory"),
+		random: {
+			...state.random,
+			handle: () => {
+				const realHandle = toFluidHandleInternal(
+					state.random.pick([
+						...Object.values(state.client.entryPoint.channels)
+							.flatMap((ca) => ca)
+							.map((c) => c.handle),
+						...Object.values(state.client.entryPoint.globalObjects)
+							.map((v) => v.handle)
+							.filter((v): v is IFluidHandle => v !== undefined),
+					]),
+				);
+				return {
+					absolutePath: realHandle.absolutePath,
+					get [fluidHandleSymbol]() {
+						return realHandle[fluidHandleSymbol];
+					},
+					async get() {
+						return realHandle.get();
+					},
+					get isAttached() {
+						return realHandle.isAttached;
+					},
+				};
+			},
+		},
+	};
+};
+
+export const DDSModelOpGenerator: AsyncGenerator<DDSModelOp, LocalServerStressState> = async (
+	state,
+) => {
+	const channelType = state.random.pick(Object.keys(state.client.entryPoint.channels));
+	const channel = state.random.pick(state.client.entryPoint.channels[channelType]);
+	const model = ddsModelMap.get(channelType);
+	assert(model !== undefined, "must have model");
+
+	const op = await model.generator(covertLocalServerStateToDdsState(state, channel));
+
+	return {
+		type: "DDSModelOp",
+		channelType,
+		channelId: channel.id,
+		op,
+	} satisfies DDSModelOp;
+};
+
+export const DDSModelOpReducer: AsyncReducer<DDSModelOp, LocalServerStressState> = async (
+	state,
+	op,
+) => {
+	const baseModel = ddsModelMap.get(op.channelType);
+	assert(baseModel !== undefined, "must have model");
+	const channel = state.client.entryPoint.channels[op.channelType].find(
+		(v) => v.id === op.channelId,
+	);
+	assert(channel !== undefined, "must have channel");
+	await baseModel.reducer(covertLocalServerStateToDdsState(state, channel), op.op as any);
+};
+
+export const validateConsistencyOfAllDDS = async (clientA: Client, clientB: Client) => {
+	const buildChannelMap = (client: Client) => {
+		const channelMap = new Map<string, IChannel>();
```

Comment:
here we build a map of all the channels in the container based on their absolute path, once we have this we can match channels in different container (clientA and clientB), and then reuse the per dds validators to ensure eventual consistency.

Reply:
This comment should be a function comment in the code